src/core/index.ts

```diff
@@ -132,32 +132,48 @@ export class CrowdinError extends Error {
  * @internal
  */
 export class CrowdinValidationError extends CrowdinError {
-    public validationCodes: { key: string; codes: string[] }[];
-    constructor(messsage: string, validationCodes: { key: string; codes: string[] }[]) {
-        super(messsage, 400);
-        this.validationCodes = validationCodes;
+    public validationCodes?: { key: string; codes: string[] }[];
+    constructor({
```

Comment:
I think we need to revert these changes and just use `CrowdinError` in condition where we have only `code`